e2etest/bgp_test.go

```diff
@@ -111,17 +130,56 @@ func setupContainers(ipv4Addresses, ipv6Addresses []string) []*frrcontainer.FRR
 
 		ibgpContainerConfig.IPv4Address = ipv4Addresses[0]
 		ibgpContainerConfig.IPv6Address = ipv6Addresses[0]
-		ebgpContainerConfig.IPv4Address = ipv4Addresses[1]
-		ebgpContainerConfig.IPv6Address = ipv6Addresses[1]
-		res, err = createFRRContainers(ibgpContainerConfig, ebgpContainerConfig)
+		multiHopGWContainerConfig.IPv4Address = ipv4Addresses[1]
+		multiHopGWContainerConfig.IPv6Address = ipv6Addresses[1]
+
+		var out string
+		out, err = executor.Host.Exec(executor.ContainerRuntime, "network", "create", multiHopNetwork, "--ipv6",
```

Comment:
Will this fail if we run the test multiple times?

Reply:
nope, there's a `network rm` in teardown.